packages/components/psammead-bulletin/src/index.stories.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import notes from '../README.md';
+import { BulletinCTA } from './index';
+
+storiesOf('Components|Bulletin', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'Call to Action',
```

Comment:
I wasn't sure whether to leave this story in to help with further development on the CTA or to just use the full container's story.